rotkehlchen/data_import/importers/uphold_transactions.py

```diff
@@ -24,14 +22,14 @@
 from rotkehlchen.types import AssetAmount, AssetMovementCategory, Fee, Location, Price, TradeType
 from rotkehlchen.utils.misc import ts_sec_to_ms
 
-logger = logging.getLogger(__name__)
-log = RotkehlchenLogsAdapter(logger)
-
 
 UPHOLD_PREFIX = 'UPH_'
 
 
 class UpholdTransactionsImporter(BaseExchangeImporter):
+    """Uphold CSV importer"""
+    name = 'Uphold'
```

Comment:
Why not do it in the constructor?

This way it's less code, but having a constructor argument at `BaseExchangeImporter` properly creates the type and lets readers of the code and type checkers know that all importers have a name and it should be set at creation time of an importer.

What you do now is hacky.